etc/profile-a-l/Beyond-All-Reason.profile

```diff
@@ -0,0 +1,42 @@
+# Firejail profile for Beyond All Reason
+# Description: Open source real time strategy game
+# This file is overwritten after every install/update
+# Persistent local customizations
+include beyondallreason.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/Beyond All Reason
+noblacklist ${HOME}/Downloads
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/Beyond All Reason
+whitelist ${HOME}/Beyond All Reason
+whitelist ${HOME}/Downloads
+
+caps.drop all
+ipc-namespace
+netfilter
+nodbus
```

Comment:
Deprecated, use `dbus-{user,system} none`.

Reply:
I've realised that there's a button that needs user level dbus, so I've restricted the system one only

Reply:
It would be good to restrict the session-bus too. I haven't found anything in BYAR source-code so it's probably a common name used by a libary. What does the button?

Reply:
The launcher has a button that just uses dbus to open the $HOME/Beyond All Reason directory in your file manager. It's not critical for running the game, but it seems to hang if permission is denied

Reply:
```
dbus-user filter
dbus-user.talk org.freedesktop.FileManager1
```